phpstan.neon

```diff
@@ -529,3 +529,6 @@ parameters:
                 - packages/FileFormatter/ValueObject/NewLine.php #15
                 - src/Application/VersionResolver.php #16
                 - utils/compiler/src/Unprefixer.php #9
+
+        # on purpose due polyfill usage
+        - '#Rule Rector\\Php73\\Rector\\BooleanOr\\IsCountableRector must implements Rector\\VersionBonding\\Contract\\MinPhpVersionInterface#'
```

Comment:
this is on purpose due polyfill usage test on https://github.com/rectorphp/rector-src/blob/main/rules-tests/Php73/Rector/BinaryOr/IsCountableRector/FixtureWithPolyfill/polyfill_function.php.inc which uses php < 7.3 check 

https://github.com/rectorphp/rector-src/blob/57a6d7af31dd2a0c38b96677c7c3e441cfd64401/rules-tests/Php73/Rector/BinaryOr/IsCountableRector/config/polyfill_config.php#L12

Reply:
I see. Polyfils were just to try experiments, but I'm not sure Rector should think of them. Those polyfils can be removed or have different behavior and its 3rd party responsibility. It might be also weird to see just few functions, but not all.

Here it breaks our contract and create unreliable behavior. I think this error should be removed a rule corrected

Reply:
Ok, I removed it and add implements MinPhpVersionInterface on IsCountableRector https://github.com/rectorphp/rector-src/pull/910/commits/45c115aa95bc11b3e38215f229e049aa099fe397